reversestring_advanced-MBA/prompt.md

```
@@ -0,0 +1,125 @@
+* La experiencia de usuario debe ser fluida e intuitiva
+
+# Estructura de entrega
+
+Entrega el código en este orden:
+
+1. Código de la Fase 1 (funcionalidad básica con botón condicional)
+
+2. Código de la Fase 2 (inversión en tiempo real)
+
+-----------------
+En este punto el chatbot me hizo varias preguntas las cuales respondí de la siguiente manera:
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)